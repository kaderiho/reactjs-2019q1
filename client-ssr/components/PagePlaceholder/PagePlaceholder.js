"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

var _react = _interopRequireDefault(require("react"));

var PagePlaceholder = function PagePlaceholder(_ref) {
  var placeholderMessage = _ref.placeholderMessage;
  return _react.default.createElement("div", {
    className: "emptyMoviesFilmList"
  }, placeholderMessage);
};

var _default = PagePlaceholder;
exports.default = _default;