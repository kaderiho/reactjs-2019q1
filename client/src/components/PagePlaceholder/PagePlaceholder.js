import React from 'react';

const PagePlaceholder = ({ placeholderMessage }) => (
    <div className="emptyMoviesFilmList">{placeholderMessage}</div>
);

export default PagePlaceholder;
