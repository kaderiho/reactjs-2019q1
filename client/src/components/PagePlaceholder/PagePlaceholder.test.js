import React from 'react';
import { shallow } from 'enzyme';

import PagePlaceholder from './PagePlaceholder';

/**
 * Test
 * 
 * - To match snapshot
 */
describe('<PagePlaceholder /> component', () => {
    it('Should render correctly', () => {
        const PagePlaceholderMessage = 'Placeholder message';
        const wrapper = shallow(<PagePlaceholder PagePlaceholderMessage={PagePlaceholderMessage}/>);

        expect(wrapper).toMatchSnapshot();
    });
});
