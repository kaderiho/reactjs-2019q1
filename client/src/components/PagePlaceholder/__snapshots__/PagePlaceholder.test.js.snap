// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PagePlaceholder /> component Should render correctly 1`] = `
<div
  className="emptyMoviesFilmList"
/>
`;
